components/datetime/src/time_zone.rs

```diff
@@ -717,10 +807,23 @@ pub(super) enum IsoSeconds {
 }
 
 /// Whether a field should be zero-padded in ISO-8601 format.
-pub(super) enum ZeroPadding {
+#[derive(Clone, Copy)]
+pub enum ZeroPadding {
     /// Add zero-padding.
     On,
 
     /// Do not add zero-padding.
     Off,
 }
+
+/// A config enum for initializing TimeZoneFormat.
+#[derive(Clone, Copy)]
+pub enum TimeZoneFormatConfig {
+    GenericNonLocationLong,                     // Pacific Time
+    GenericNonLocationShort,                    // PT
+    GenericLocation,                            // Los Angeles Time
+    SpecificNonLocationLong,                    // Pacific Standard Time
+    SpecificNonLocationShort,                   // PDT
+    LocalizedGMT,                               // GMT-07:00
+    Iso8601(IsoFormat, IsoMinutes, IsoSeconds), // -07:00
```

Comment:
> praise

Good job adding the `IsoMinutes` and `IsoSeconds` here too. I didn't remember or think about adding those when I was drafting this.